prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -246,3 +246,7 @@ Component "breakout.{{ .Env.XMPP_DOMAIN }}" "muc"
     muc_room_locking = false
     muc_room_default_public_jids = true
 {{ end }}
+
+{{ if .Env.XMPP_RUN_AS_ROOT }}
+run_as_root = true
```

Comment:
Why is it necessary to run prosody as root if we expect the container to run with an UID different to 0 ?

Reply:
I didn't end up using this, I will remove it. I was trying to see of letting in run in root would be an easier solution within k8s than setting up a securityContext that may have to be disabled when deployed to OpenShift, proved unnecessary